magnolia-voting-app/src/main/resources/voting-contentApp/webresources/css/submitPage.css

```diff
@@ -0,0 +1,28 @@
+.parent{
+    background-color: #E3E2E2;
+    width: 700px;
+    height: 265px;
+    margin: 0 auto;
+}
+
+.img {
+    float: left;
+    width: 200px;
+    height: 200px;
+    padding: 30px;
+}
+h1 {
+    color: #4EC32F;
+    padding-top: 50px;
+    font-size: 35px;
+}
+.thanks {
```

Comment:
try to have meaningful name, what is "thanks" here? thanks, message container? or thanks-content?